variantgrid/settings/env/shariant.py

```diff
@@ -260,3 +260,6 @@
 VIEW_GENE_HOTSPOT_GRAPH = False
 
 GENE_RELATION_PANEL_APP_LIVE_UPDATE = True
+
+# if None lab record ID will be used for records instead of CR_lab_id
+VARIANT_CLASSIFICATION_ID_OVERRIDE_PREFIX = 'SHARIANT'
```

Comment:
Simpler to make the value a boolean, Id's set it to False in default_settings and True in shariant.py